tests/app/db/test_db_init.py

```diff
@@ -68,8 +78,10 @@ def test_get_db_session_failure() -> None:
 
 
 @patch('app.db.db_init.async_scoped_session', return_value=Mock(spec=async_scoped_session))
-@patch('app.db.db_init._engine_read', Mock(spec=AsyncEngine))
-@patch('app.db.db_init._engine_write', Mock(spec=AsyncEngine))
+@patch('app.db.db_init._engine_enp_read', Mock(spec=AsyncEngine))
+@patch('app.db.db_init._engine_enp_write', Mock(spec=AsyncEngine))
+@patch('app.db.db_init._engine_napi_read', Mock(spec=AsyncEngine))
+@patch('app.db.db_init._engine_napi_write', Mock(spec=AsyncEngine))
```

Comment:
Ditto implementation test.  I don't see how this is really testing anything useful.